pyramid/tests/test_config/test_security.py

```diff
@@ -108,14 +108,18 @@ def test_set_default_csrf_options(self):
         self.assertEqual(result.header, 'X-CSRF-Token')
         self.assertEqual(list(sorted(result.safe_methods)),
                          ['GET', 'HEAD', 'OPTIONS', 'TRACE'])
+        self.assertTrue(result.callback is None)
 
     def test_changing_set_default_csrf_options(self):
         from pyramid.interfaces import IDefaultCSRFOptions
         config = self._makeOne(autocommit=True)
+        def callback(request): return True
```

Comment:
`callback = lambda x: True`

gives pep8 warnings, since you aren't supposed to assign a lambda to a variable...